distro/main.go

```diff
@@ -40,7 +40,7 @@ var (
 			},
 			Namespace: []string{
 				"https://storage.googleapis.com/riff-releases/latest/riff-build.yaml",
-				"https://storage.googleapis.com/riff-releases/riff-cnb-buildtemplate-0.1.0.yaml",
+				"https://storage.googleapis.com/projectriff/riff-buildtemplate/riff-cnb-buildtemplate-0.0.1-snapshot-ci-64830c3bbc6503beafdae382ead115806fa100ca.yaml",
```

Comment:
What will stop us shipping this snapshot as part of a release?

Reply:
Including a snapshot in `stable` could cause problems if it's difficult to rebuild at the correct version (which would make it hard to apply a point fix for a bug without getting a load of other changes that are not necessarily desirable).

Including a snapshot in `latest` does seem counter-intuitive as you point out. I guess it depends on the definition of `latest` - does that just apply to knative components or is it meant to include riff components too?

Reply:
It's a *pinned* snapshot (as seen in the commit sha that is part of the name).
I'm fine switching it to a "tracking" snapshot for now (and then we can pin / release it when we release `riff`)

Reply:
I agree with Eric, it's no different than the pinned snapshot builds we use for knative.